Style-guide/README.md

```diff
@@ -81,39 +78,24 @@ Descriptive and consistent naming makes software easier to read and understand.
 - naming methods for their side effects
   - verb methods follow the -ed, -ing rule for the non-mutating version
   - noun methods follow the formX rule for the mutating version
-  - boolean types should read like assertions
+  - boolean types should read like assertions (`isLoading`, `isHidden`, etc)
   - protocols that describe _what something is_ should read as nouns
   - protocols that describe _a capability_ should end in _-able_ or _-ible_
 - using terms that don't surprise experts or confuse beginners
-- generally avoiding abbreviations
-- using precedent for names
+- generally avoiding abbreviations // a few well-known ones are allowed, such as, `Html` and `Id`. Any others?
+- using precedent for names // ???
```

Comment:
(I've had to change the line above to trigger a diff, else we couldn't comment on the line. The inline comment will be removed.)

Does anyone understand what this is about?

Reply:
Another instance where Swift Guidelines link would be helpful. I think it's ok to keep this list of takeaways from Swift Guidelines succinct as long as we provide a link, so there's a reference for detailed explanation.
Otherwise we would just be duplicating swift.org content.